js_modules/dagster-ui/packages/ui-core/src/graph/asyncGraphLayout.ts

```diff
@@ -63,7 +63,9 @@ const _assetLayoutCacheKey = (graphData: GraphData, opts: LayoutAssetGraphOption
   return `${JSON.stringify(opts)}${JSON.stringify({
     downstream: recreateObjectWithKeysSorted(graphData.downstream),
     upstream: recreateObjectWithKeysSorted(graphData.upstream),
-    nodes: Object.keys(graphData.nodes).sort(),
+    nodes: Object.keys(graphData.nodes)
+      .sort()
+      .map((key) => graphData.nodes[key]),
```

Comment:
Hmm I'm not sure if going all the way to the full objects is a worth it offhand, this could make the cache keys a few megabytes for large graphs?  As long as we don't see any serialization performance issues or anything like that, seems fine!

Reply:
discussed offline - The string is hashed so the keys are short